### DISCUSSION TITLE
Dynamic learning rate

### DISCUSSION THREAD

**Discussion by ANAS1997AA** [ultralytics/ultralytics#8871] [General]
can anyone give me the code to train a yolov8n model on a dynamic learning rate

**Comment by pderrenger** [comment]
Certainly! To train a YOLOv8 model with a dynamic learning rate, you can utilize the `lr0` (initial learning rate) and `lrf` (final learning rate factor) parameters in your training command. Here's a quick example:

```bash
yolo detect train data=coco128.yaml model=yolov8n.yaml epochs=100 lr0=0.01 lrf=0.1
```

In this example, `lr0=0.01` sets the initial learning rate, and `lrf=0.1` means the learning rate will decrease to 10% of the initial rate by the end of training. Adjust these values as needed for your specific requirements. Happy training! 😊
